src/ui/components/Badge/styles.scss

```diff
@@ -9,11 +9,17 @@
   margin: 0 0 6px;
 
   @include respond-to(medium) {
+    display: flex;
+    flex-direction: row-reverse;
     margin: 0 0 12px;
   }
 
   .Icon {
-    @include margin-start(6px);
+    @include margin(0, 6px, 0, 0);
```

Comment:
I didn't know we had this function. I'm guessing it just inverts the left/right for RTL?

Reply:
Hah, I see it is. Cool!